options/locale/locale_en-US.ini

```diff
@@ -192,7 +192,7 @@ platform_desc = Gitea runs anywhere <a target="_blank" rel="noopener noreferrer"
 lightweight = Lightweight
 lightweight_desc = Gitea has low minimal requirements and can run on an inexpensive Raspberry Pi. Save your machine energy!
 license = Open Source
-license_desc = Go get <a target="_blank" rel="noopener noreferrer" href="https://code.gitea.io/gitea">code.gitea.io/gitea</a>! Join us by <a target="_blank" rel="noopener noreferrer" href="https://github.com/go-gitea/gitea">contributing</a> to make this project even better. Don't be shy to be a contributor!
+license_desc = "Go get <a target="_blank" rel="noopener noreferrer" href="https://code.gitea.io/gitea">code.gitea.io/gitea</a>! Join us by <a target="_blank" rel="noopener noreferrer" href="https://github.com/go-gitea/gitea">contributing</a> to make this project even better. Don't be shy to be a contributor!"
```

Comment:
use ` ?

Reply:
No, don't use it. It has already caused many issues.

Reply:
So maybe this change is unnecessary.